### ISSUE TITLE
[Accessibility]: DiffStat has insufficient color contrast

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#34541]
### Audit type

ARC Toolkit

### Problem description

<img width="181" alt="CleanShot 2022-04-26 at 22 50 04@2x" src="https://user-images.githubusercontent.com/19534377/165389934-7334558e-2384-4d2e-a4ac-ab514bbc4bf4.png">

The green color here doesn't have a high enough contrast on either body-bg or white.

### Expected behavior

The color contrast check passes.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.

**Comment by umpox** [comment]
Looking into this. I have a feeling that this is one example where this would pass using the WCAG 3.0 color contrast algorithm. If so, want to discuss further with TPGi if these are essential to change
